pybites_tools/meaning.py

```diff
@@ -7,14 +7,12 @@
 
 def get_meaning(word, site, datasrc):
 
-    try:
-        response = requests.get(site + word)
-        response.raise_for_status()
-    except HTTPError as http_err:
-        print(f"HTTP error occurred: {http_err}")
-        exit(255)
-    except Exception as err:
-        print(f"Other error occurred: {err}")
+    response = requests.get(site + word)
+    if not response.ok:
+        if response.status_code == 404:
```

Comment:
Weren't we going to use `response.reason` here? 

In that case you don't need the if/else, you can just do: `f"An error occurred - status code: {response.status_code} / reason: {response.reason}"`

Reply:
So, I missed reason off by accident. Basic thinking was, if its 404 that means it cant find the word so lets call that explicitly but then we might get other errors and we should show that. Just pushed a new fix

Reply:
OK I do like the specific 404 check, thanks for adding reason in the else.